src/reducers-actions/pendingOrders.ts

```diff
@@ -0,0 +1,79 @@
+import { PendingTxObj } from 'api/exchange/ExchangeApi'
+import { GP_ORDER_TX_HASHES } from 'const'
+
+export const enum ActionTypes {
+  SAVE_PENDING_ORDERS = 'SAVE_PENDING_ORDERS',
+  REMOVE_PENDING_ORDERS = 'REMOVE_PENDING_ORDERS',
+}
+
+interface Actions<T> {
+  type: ActionTypes
+  payload: T
+}
+
+export const savePendingOrdersAction = (payload: {
+  orders: PendingTxObj
+  networkId: number
+}): Actions<{
+  orders: PendingTxObj
+  networkId: number
+}> => ({
```

Comment:
As much as I don't like several interfaces scattered about (looking at you, ExchangeApi!) it makes the function signature more readable. Could you do that here as well?

Reply:
you mean making `PendingOrderInterface` instead of 

```js
{
  orders: PendingTxObj
  networkId: number
}
``` 
?

Reply:
Yep!